c/sfntdiff/source/Dmain.c

```diff
@@ -152,12 +152,6 @@ IntN main(IntN argc, Byte8 *argv[]) {
 
         /* See if we can recognize the file type */
         value = fileSniff(1);
```

Comment:
Please add code to confirm that file 2 is a valid filetype -- see all three instance of `switch (value)` in `Dmain.c`.

Currently if you'd get this for an invalid first file:
```
$ sfntdiff LICENSE.txt OTF/SourceCodePro-Regular.otf 
Thu Mar 14 09:59:02 2019
sfntdiff (2.21217) (-d 0)  files:
< LICENSE.txt
> OTF/SourceCodePro-Regular.otf
sfntdiff [WARNING]: unsupported/bad file [LICENSE.txt] (ignored)
```
versus this for an invalid second file:
```
$ sfntdiff OTF/SourceCodePro-Regular.otf LICENSE.txt 
Thu Mar 14 09:59:05 2019
sfntdiff (2.21217) (-d 0)  files:
< OTF/SourceCodePro-Regular.otf
> LICENSE.txt
sfntdiff [FATAL]: file error <premature EOF> [LICENSE.txt]
```

Reply:
Done. Please have another look. Thanks!